src/components/common.js

```diff
@@ -534,7 +537,13 @@ export const HeroWrapper = ({ showMenu = true, bigSubhead = false, children }) =
         heavyWrapper('access data'), ', ', heavyWrapper('run analysis tools'), ', ',
         span({ style: { whiteSpace: 'nowrap' } }, ['and', strong([' collaborate'])]),
```

Comment:
@kyuksel the usage of `strong` here instead of `heavyWrapper` is causing this styling issue:

![image](https://user-images.githubusercontent.com/484484/173919211-21864d24-88d3-4d4f-aa38-3d39f42441b3.png)

Reply:
Thanks for the catch!

@rtitle would you be able to revert the change in your PR or shall I make a separate PR?

Reply:
@kyuksel Rob didn't introduce this issue. You can see it in dev. The issue is that in one code path (the one we were testing apparently) `heavyWrapper` makes things bold/strong, but in the other it does not. I would say we should simplify our lives and just always go with bold.

![image](https://user-images.githubusercontent.com/484484/173921925-ed36f549-1b64-4880-8dc6-be105f3c243d.png)

Reply:
I can change it back to `heavyWrapper`. (I thought **collaborate** was supposed to be emphasized 😂 )

Reply:
I didn't mean to imply Rob introduced the issue. Reverting my change would be a small one and this PR seems close to being merged so I thought it could piggy-back on this PR but I can make a separate one if it'll complicate things.

> I would say we should simplify our lives and just always go with bold.

I don't feel comfortable changing it without knowing why it was `bold`ed in one case and not the other in the first place.

> I think you were trying to avoid wrapping CSS

Wrapping would be okay. I mainly thought it was unnecessary but missed the `bigSubHead = false` case.

Reply:
Done

Reply:
Thank you!